plugins/email_service/app/views/email_service/domain_verifications/show.html.haml

```diff
@@ -1,62 +1,135 @@
-            =@verified_identity[:status]
+            %strong verification_status:
+          %td
+            =@verified_domain[:verification_status]     
         %tr
           %td
-            %strong DKIM Tokens:
-          %td 
-            =@verified_identity[:dkim_tokens]
+            %strong feedback_forwarding_status:
+          %td
+            =@verified_domain[:feedback_forwarding_status]
         %tr
           %td
-            %strong DKIM Enabled:
-          %td 
-            =@verified_identity[:dkim_enabled]
+            %strong Verified_for_sending_status:
+          %td
+            =@verified_domain[:verified_for_sending_status]
+        - if @verified_domain[:dkim_attributes]
+          %tr
+            %td
+              %strong dkim_attributes:
+            %td
+              =@verified_domain[:dkim_attributes].inspect
+          %tr
+            %td
+              %strong status:
+            %td
+              =@verified_domain[:dkim_attributes][:status]
+          %tr
+            %td
+              %strong tokens:
+            %td
+              =@verified_domain[:dkim_attributes][:tokens]
+          %tr
+            %td
+              %strong signing_attributes_origin:
+            %td
+              =@verified_domain[:dkim_attributes][:signing_attributes_origin]
+          %tr
+            %td
+              %strong current_signing_key_length:
+            %td
+              =@verified_domain[:dkim_attributes][:current_signing_key_length].inspect
+          %tr
+            %td
+              %strong next_signing_key_length:
+            %td
+              =@verified_domain[:dkim_attributes][:next_signing_key_length].inspect
+          %tr
+            %td
+              %strong last_key_generation_timestamp:
+            %td
+              =@verified_domain[:dkim_attributes][:last_key_generation_timestamp].inspect
         %tr
           %td
-            %strong DKIM Verification Status:
-          %td 
-            =@verified_identity[:dkim_verification_status]
-
-%hr
-- if @verified_identity[:dkim_tokens]
-  Create the following CNAME DNS records.
+            %strong mail_from_attributes:
+          %td
+            =@verified_domain[:mail_from_attributes].inspect
+        %tr
+          %td
+            %strong policies:
+          %td
+            =@verified_domain[:policies]
+        %tr
+          %td
+            %strong tags:
+          %td
+            =@verified_domain[:tags].inspect
+        - if @verified_domain[:configuration_set_name]
+          %tr
+            %td
+              %strong configuration_set_name:
+            %td
+              =@verified_domain[:configuration_set_name]
+          %hr
+        -#
+          %tr
+            %td
+              %strong Verification Token:
+              %a{ href: "#", title: "You need to create a TXT record for your domain using this token displayed here.", data: { toggle: "tooltip", html: true} }
+                %i.fa.fa-question-circle-o
```

Comment:
Please remove not used code. It is hard to see what is going on in the code otherwise

Reply:
removed.